src/misc/cross.cpp

```diff
@@ -439,6 +440,224 @@ FILE *fopen_wrap_ro_fallback(const std::string &filename, bool &is_readonly)
 	return fp;
 }
 
+bool wild_match(const char *haystack, char *needle)
+{
+	size_t max, i;
```

Comment:
Please put each new variable on its own line.
Those follow on variables are easy to miss when quickly reading a function.

Reply:
Thanks for the explanation. Done.